src/Components/Confirmation/ConfirmationSteps.tsx

```diff
@@ -0,0 +1,589 @@
+          value={translateNumber(String(member.birthMonth).padStart(2, '0')) + '/' + translateNumber(member.birthYear)}
+        />
+        <ConfirmationItem
+          label={
+            <FormattedMessage id="confirmation.headOfHouseholdDataBlock-conditionsText" defaultMessage="Conditions:" />
+          }
+          value={conditionsString(member.conditions)}
+        />
+        <ConfirmationItem
+          label={<FormattedMessage id="confirmation.headOfHouseholdDataBlock-incomeLabel" defaultMessage="Income:" />}
+          value={
+            hasIncome && incomeStreams.length > 0 ? (
+              <ul>{listAllIncomeStreams(incomeStreams)}</ul>
+            ) : (
+              <FormattedMessage id="confirmation.noIncome" defaultMessage="None" />
+            )
+          }
+        />
+        <ConfirmationItem
+          label={
+            <FormattedMessage
+              id="confirmation.headOfHouseholdDataBlock-healthInsuranceText"
+              defaultMessage="Health Insurance: "
+            />
+          }
+          value={displayHealthInsurance()}
+        />
+      </ConfirmationBlock>
+    );
+  });
+
+  return <>{householdMemberDataBlocks}</>;
+}
+
+function Expenses() {
+  const { formData } = useContext(Context);
+  const { formatMessage } = useIntl();
+  const translateNumber = useTranslateNumber();
+  const expenseOptions = useConfig<FormattedMessageMap>('expense_options');
+
+  const editExpensesAriaLabel = {
+    id: 'confirmation.expense.edit-AL',
+    defaultMessage: 'edit expenses',
+  };
+  const expensesIconAlt = {
+    id: 'confirmation.expense.icon-AL',
+    defaultMessage: 'expenses',
+  };
+
+  const allExpenses = () => {
+    if (formData.expenses.length === 0) {
+      return <FormattedMessage id="confirmation.none" defaultMessage="None" />;
+    }
+    const mappedExpenses = formData.expenses.map((expense, i) => {
+      return (
+        <ConfirmationItem
+          label={<>{expenseOptions[expense.expenseSourceName]}:</>}
+          value={translateNumber(formatToUSD(Number(expense.expenseAmount)))}
+          key={i}
+        />
+      );
+    });
+
+    return mappedExpenses;
+  };
+
+  return (
+    <ConfirmationBlock
+      icon={<Expense title={formatMessage(expensesIconAlt)} />}
+      title={
+        <FormattedMessage
+          id="confirmation.headOfHouseholdDataBlock-expensesLabel"
+          defaultMessage="Monthly Household Expenses"
+        />
+      }
+      editAriaLabel={editExpensesAriaLabel}
+      stepName="hasExpenses"
+    >
+      {allExpenses()}
+    </ConfirmationBlock>
+  );
+}
+
+function Assets() {
+  const { formData } = useContext(Context);
+  const { formatMessage } = useIntl();
+  const translateNumber = useTranslateNumber();
+
+  const editAssetsAriaLabel = {
+    id: 'confirmation.assets.edit-AL',
+    defaultMessage: 'edit assets',
+  };
+  const assetsIconAlt = {
+    id: 'confirmation.expense.icon-AL',
+    defaultMessage: 'assets',
+  };
```

Comment:
Added to translations config.

Reply:
@CalebPena The assets icon is using the expense icon's alt, is this intentional?

Reply:
Nope, that is not intentional. Updating now.